src/components/Login.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import '../assets/scss/Login.scss';
+import '../assets/scss/SocialLogin.scss';
+
+import FacebookOption from './social/facebook';
+import GoogleOption from './social/google';
+
+class Login extends React.Component {
+  render() {
```

Comment:
I would prefer you use function-based component for this Login component.
since it favors best practices.
https://medium.com/@Zwenza/functional-vs-class-components-in-react-231e3fbd7108

Reply:
@reifred thankyou for the resource